src/cmd-buildextend-metal

```diff
@@ -114,38 +114,29 @@ if [ "${rev_parsed}" != "${commit}" ]; then
 fi
 
 
-itype=metal
-img=${name}-${build}-${itype}.raw
+img=${name}-${build}-metal.raw
 if [ -f "${builddir}/${img}" ]; then
     echo "Bare metal image already exists"
     exit
 fi
 
 path=${PWD}/${img}
-# for anaconda logs
-mkdir -p tmp/anaconda
-
-if [ "$arch" == "aarch64" ]; then
-    run_virtinstall "${ostree_repo}" "${ref}" "${path}.tmp" --variant="${itype}-uefi"
-    /usr/lib/coreos-assembler/gf-platformid "${path}"{.tmp,} metal
-    rm -f "${path}".tmp
-else
-    if [ ! -e "$PWD/tmp/ostree-size.json" ]; then
-        echo "Estimating disk size..."
-        /usr/lib/coreos-assembler/estimate-commit-disk-size --repo "$ostree_repo" "$ref" > "$PWD/tmp/ostree-size.json"
-    fi
-    size="$(jq '."estimate-mb".final' "$PWD/tmp/ostree-size.json")"
-    # 384M is the non-ostree partitions
-    size="$(( size + 384 ))M"
-    echo "Disk size estimated to $size"
-    kargs="$(python3 -c 'import sys, yaml; args = yaml.load(sys.stdin)["extra-kargs"]; print(" ".join(args))' < "$configdir/image.yaml")"
-    kargs="$kargs console=tty0 console=${VM_TERMINAL},115200n8 ignition.platform.id=metal"
-
-    qemu-img create -f qcow2 "${path}.qcow2" "$size"
-    runvm_with_disk "${path}.qcow2" /usr/lib/coreos-assembler/create_disk.sh /dev/vda "$ostree_repo" "${ref-:${commit}}" /usr/lib/coreos-assembler/grub.cfg "$name" "\"$kargs\""
-    qemu-img convert "${path}.qcow2" "$path"
-    rm "${path}.qcow2"
+
+if [ ! -e "$PWD/tmp/ostree-size.json" ]; then
```

Comment:
This made sense before when UEFI and BIOS were separate images, but now that they're unified, there's really no point in this conditional.